en-US/ranking/process.md

```diff
@@ -75,10 +75,19 @@ Supervisors are also not allowed to vote for their own mapsets. They must go thr
 
 If your mapset is denied, you will receive a comment from one or more Ranking Supervisors with the reason(s) why. After this period, you are eligible to update your map with fixes to submit again for rank. **Keep in mind that if denied, your set will be placed in a cooldown period for 3 days until you are able to submit it again.**
 
-### 5b. Mapset Blacklisted
+### 5b. Mapset On-Hold
+
+When a mapset is put into this state, its position in the queue is retained but is awaiting further minor changes that needs to be resolved. This is used to put mapsets that are of otherwise rankable quality that is lacking in formalities which would otherwise be grounds for denial, such as improper metadata or overlapping notes, into a state where all supervisor votes are reset and bars the mapset from any further voting by supervisors, as well as keeping its queue position. 
```

Comment:
```suggestion
This is used to put mapsets that are of otherwise rankable quality that is lacking in formalities which would otherwise be grounds for denial, such as improper metadata or overlapping notes, into a state where all supervisor votes are reset and bars the mapset from any further voting by supervisors, as well as keeping its queue position. 
```